packages/main/src/themes/Bar.css

```diff
@@ -0,0 +1,52 @@
+:host {
+	background-color: var( --sapPageHeader_Background);
+	height: var(--_ui5_bar_base_height);
+	width: 100%;
+}
+
+.ui5-bar {
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	height: inherit;
+	width: inherit;
+	background-color: inherit;
+	margin-bottom: 0.5rem;
+	margin-top: 0.5rem;
+}
+
+.ui5-bar .ui5-bar-leftcontent-container {
+	padding-left: 0.5rem;
+}
+
+.ui5-bar .ui5-bar-rightcontent-container {
+	padding-right: 0.5rem;
+}
+.ui5-bar .ui5-bar-midcontent-container {
+	padding-left: 0.5rem;
+	padding-right: 0.5rem;
+}
+
+.ui5-bar.ui5-bar-design-bottomborder{
+	background-color: var(--sapPageHeader_Background);
+	box-shadow: inset 0 -0.0625rem var(--sapPageFooter_BorderColor);
+}
+
+.ui5-bar.ui5-bar-design-topborder{
+	background-color: var(--sapPageFooter_Background);
+	border-top: 0.0625rem solid var(--sapPageFooter_BorderColor);
+}
+
+.ui5-bar.ui5-bar-design-allborders{
+	border-radius: var(--sapElement_BorderCornerRadius);
+	background-color: var(--sapPageFooter_Background);
+	box-shadow: var(--sapContent_Shadow1);
+	border: none;
+}
+
+::slotted([slot="middleContent"]),
+::slotted([slot="endContent"]),
+::slotted([slot="startContent"]){
+	display: flex;
```

Comment:
Not sure about this part: I would expect that most people will directly place buttons in the left/mid/right slots and expect them to go to the left/mid/right automatically. This CSS assumes that people will always place a single div inside each slot and put buttons in the slot.

@ilhan007 @fifoosid @pskelin Opinioins?

Reply:
Both cases are possible, but yes I expect adding direct children what most of the users will do

```html
<ui5-bar
   <ui5-button slot="startContent">
   <ui5-button slot="startContent">
   <ui5-title  slot="midContent">
   <ui5-button slot="endContent"
   <ui5-button slot="endContent"
```

Reply:
I will just implement both cases. And also will create samples for both cases.

Reply:
Isn't it  `::slotted(*)` the same selector only shorter? Didn't see slot contents that doesn't need does properties.